package.js

```diff
@@ -8,27 +8,24 @@
 
 Package.describe({
   name: 'useraccounts:flow-routing',
-  summary: 'UserAccounts package providing routes configuration capability via kadira:flow-router.',
-  version: '1.15.0',
+  summary: 'UserAccounts package providing routes configuration capability via ostrio:flow-router-extra.',
+  version: '1.16.0',
```

Comment:
This change is going to break the apps using kadira:flow-router, right?

If so I would publish it as 2.0.0, what do you think?

Reply:
Yes, you are right. I'm changing to version 2.0.0.

Reply:
Changes done. Go ahead @filipenevola .